app/views/api/v1/users/_user.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.id user.id
```

Comment:
Why do you need a JSON builder? What do you think about using serializer instead of JSON builder?

Reply:
It was just easier to start with :)

Reply:
👌🏻